src/hooks/useKeyboardNavigation.js

```diff
@@ -27,11 +29,68 @@ const useKeyboardNavigation = ({
 
   useEffect(() => {
     return tinykeys(emojiListRef.current, {
-      ArrowRight: focusNextEmoji,
-      ArrowLeft: focusPrevEmoji,
+      ArrowRight: () => navigateGrid(RIGHT),
+      ArrowLeft: () => navigateGrid(LEFT),
+      ArrowUp: () => navigateGrid(UP),
+      ArrowDown: () => navigateGrid(DOWN),
     });
   }, []);
 
+  const updateActiveItem = newActiveItem => {
+    if (newActiveItem && newActiveItem.firstChild)
+      newActiveItem.firstChild.focus();
```

Comment:
We're using focus quite a lot. I think it's wise to create a general use case
`focusElement()` function that will also safely try/catch, handle null values and other edge cases.